x/ethbridge/querier/querier.go

```diff
@@ -35,13 +35,13 @@ func queryEthProphecy(ctx sdk.Context, cdc *codec.Codec, req abci.RequestQuery,
 	if err != nil {
 		return []byte{}, sdk.ErrInternal(sdk.AppendMsgToErr("failed to parse params: %s", err.Error()))
 	}
-	id := strconv.Itoa(params.Nonce) + params.EthereumSender.String()
+	id := strconv.Itoa(params.ChainID) + strconv.Itoa(params.Nonce) + params.EthereumSender.String()
```

Comment:
so we're doing `int -> string` here for the ChainID again which shouldn't be necessary

Reply:
see comment above